docs/guides/component-testing/faq.mdx

```diff
@@ -106,7 +106,7 @@ component, just like in the application. See our guide on
 [Styling Components](/guides/component-testing/styling-components) for more
 information on doing so.
 
-### Why doesn't my spec show in the spec explorer?
+### Why doesn't my spec show in the Specs page?
```

Comment:
@pstakoun Did you already do a CMD+F for other instances of "spec(s) explorer"? I don't think there were many.

Reply:
I just did a check and the only things I could find where the name of image files or in the change log.

<img width="790" alt="image" src="https://user-images.githubusercontent.com/1702361/215135065-8bb83212-5e02-4d65-b9a0-140a06221473.png">

Reply:
Yeah I checked as well and caught a couple, didn't see any more